### PR TITLE
ticdc: add docs about ticdc open api

### PR DESCRIPTION
**Pull request by asddongmen**
### What is changed, added or deleted? (Required)
Add docs about the new Open APIs of TiCDC.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

 [Develop openapi for ticdc. #2411](https://github.com/pingcap/ticdc/issues/2411)
 [*: add some open API to cdc. #2416](https://github.com/pingcap/ticdc/pull/2416)
 [http_*: add create and update changefeed API #2472](https://github.com/pingcap/ticdc/pull/2472)
### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by asddongmen** [comment]
@hi-rustin @amyangfei @leoppro @ben1009  please help to review this PR. Thanks~!!!!

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -540,90 +540,541 @@ cdc cli changefeed resume -c test-cf --pd=http://10.0.10.25:2379
...
+    "id": "c6a43c16-0717-45af-afd6-8b3e01e44f5d",
+    "pid": 25432,
+    "is_owner": true
 }
 ```
 
-### 驱逐 owner 节点
+### 检查 TiCDC 集群的健康状态 
+
+该接口是一个同步接口，在集群健康的时候会返回 `200 OK`。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:573]: 修改意见

Comment (TomShawn) [inline ticdc/manage-ticdc.md:573]:
```suggestion
该接口是一个同步，在集群健康的时候会返回 `200 OK`。
```

**Comment by asddongmen** [comment]
This pr is updated in #6846

**Pull request closed without merging**